File: prime_generator.py
import sys, math	

def primecheck(a):
	from math import sqrt
	if (a==2):
		return 2
	elif (a>2):
		flag=0
		for i in range(2,int(sqrt(a)+1)):
			if(a%i==0):
				flag=1
				break
			i+=1
		if(flag==1):
			return 0
		else:
			return a
def main():
	from sys import stdin,stdout
	t =map(int,sys.stdin.readline().split())
	t = t[0]
	x = []
	while(t):
		a,b = map(int, sys.stdin.readline().split(' '))
		# a = int(a)
		b = int(b)
		if(b>=1 and a>=1 and a<=b):
			for p in range(a,b+1):
				if(primecheck(p)):
					x.append(p)
		x.append("\n")
		t-=1
	for c in x:
		print c

if __name__ == "__main__":
    main()

File: prime_generator.c
#include<stdio.h>
#include<math.h>
int primecheck(long long  int a)
{
	if (a==2)
		return 2;
	else if(a>2)
	{
		int flag=0,i;
		for(i=2; i<=sqrt(a); i++)
		{
			if(a%i==0)
				{
					flag=1;
					break;
				}
		}
		if(flag==0)
		{
			return a;
		}
		else 
		{
			return 0;
		}
	}
	else 
		return 0;
}
// using namespace std;
int main()
{
	int t,i,j,p=0,q;
	scanf("%d",&t);
	long int x[100][t];
	while(t--)
	{
		q=0;
		long long int a,b;
		scanf("%lld %lld",&a,&b);
		for(i=a;i<=b;i++)
		{
			// if(a>=1 && b>=1 && b<=1000000000 && a<=1000000000 && b-a<=100000)
			if(primecheck(i))
			{
				x[p][q]=i;
				q++;
			}
		}
	p++;
	}
	for(i=0;i<p;i++)
	{
		for(j=0;j<q;j++)
		{
			printf("%ld",x[i][j]);
			printf("\n");
		}
		printf("\n");
	}
	return 0;
}
